From 48ed371cc27420afd2fb4e5c8bb7d47f7824d32c Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 2 Jun 2017 16:24:34 +0800
Subject: [PATCH 1906/5242] MLK-15001-2 arm64: fsl-imx8qm.dtsi: Remove imxdpu
 nodes

commit  492cec006ba74bca642b904cb52c291018a705f8 from
https://source.codeaurora.org/external/imx/linux-imx.git

No driver works with the imxdpu nodes, so let's remove them.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    8 -----
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi      |   32 --------------------
 2 files changed, 40 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 238a859..7e49216 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -310,14 +310,6 @@
 	status = "okay";
 };
 
-&imxdpu0 {
-	status = "okay";
-};
-
-&imxdpu1 {
-	status = "okay";
-};
-
 &lvds0 {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index a662b0d..f1a8d5c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1019,38 +1019,6 @@
 		status = "disabled";
 	};
 
-	imxdpu0: imxdpu0@0x56180000 {
-		compatible = "fsl,imx8qm-imxdpuv1";
-		reg = <0x0 0x56000000 0x0 0x1000000>;
-		interrupts = <0 40 0x4>, <0 41 0x4>, <0 42 0x4>,
-			<0 43 0x4>, <0 44 0x4>, <0 45 0x4>, <0 46 0x4>,
-			<0 47 0x4>, <0 48 0x4>, <0 49 0x4>, <0 50 0x4>;
-		interrupt-names = "irq_grp00", "irq_grp01", "irq_grp02",
-			"irq_grp03", "irq_grp04", "irq_grp05", "irq_grp06",
-			"irq_grp07", "irq_grp08", "irq_grp09", "irq_grp10";
-		clocks = <&clk IMX8QM_DC0_PLL0_CLK>, <&clk IMX8QM_DC0_PLL1_CLK>,
-			<&clk IMX8QM_DC0_DISP0_CLK>, <&clk IMX8QM_DC0_DISP1_CLK>;
-		clock-names = "clk_pll0", "clk_pll1", "clk_disp0", "clk_disp1";
-		power-domains = <&pd_dc0>;
-		status = "disabled";
-	};
-
-	imxdpu1:  imxdpu1@0x57180000 {
-		compatible = "fsl,imx8qm-imxdpuv1";
-		reg = <0x0 0x57000000 0x0 0x1000000>;
-		interrupts = <0 152 0x4>, <0 153 0x4>, <0 154 0x4>,
-			<0 155 0x4>, <0 156 0x4>, <0 157 0x4>, <0 158 0x4>,
-			<0 159 0x4>, <0 160 0x4>, <0 161 0x4>,<0 162 0x4>;
-		interrupt-names = "irq_grp00", "irq_grp01", "irq_grp02",
-			"irq_grp03", "irq_grp04", "irq_grp05", "irq_grp06",
-			"irq_grp07", "irq_grp08", "irq_grp09", "irq_grp10";
-		clocks = <&clk IMX8QM_DC1_PLL0_CLK>, <&clk IMX8QM_DC1_PLL1_CLK>,
-			<&clk IMX8QM_DC1_DISP0_CLK>, <&clk IMX8QM_DC1_DISP1_CLK>;
-		clock-names = "clk_pll0", "clk_pll1", "clk_disp0", "clk_disp1";
-		power-domains = <&pd_dc1>;
-		status = "disabled";
-	};
-
 	lvds0: lvds@56241000 {
 		compatible = "fsl,imx8qm-lvds";
 		reg = <0x0 0x56241000 0x0 0x1000>;
-- 
1.7.9.5

